src/server/game/Entities/Player/Player.cpp

```diff
@@ -12933,20 +12933,7 @@ void Player::RemoveItem(uint8 bag, uint8 slot, bool update, bool swap)
 
                     // remove held enchantments, update expertise
                     if (slot == EQUIPMENT_SLOT_MAINHAND)
-                    {
-                        if (pItem->GetItemSuffixFactor())
-                        {
-                            pItem->ClearEnchantment(PROP_ENCHANTMENT_SLOT_3);
-                            pItem->ClearEnchantment(PROP_ENCHANTMENT_SLOT_4);
-                        }
-                        else
-                        {
-                            pItem->ClearEnchantment(PROP_ENCHANTMENT_SLOT_0);
-                            pItem->ClearEnchantment(PROP_ENCHANTMENT_SLOT_1);
-                        }
```

Comment:
This code block is useless, I don't get why random stats should be removed on specific slots for main hand weapon only.
Plus, according to the old code (before this PR), PROP_ENCHANTMENT_SLOT_3 & 4 _(eq :  PROP_ENCHANTMENT_SLOT_0 & 1)_ are **NEVER** used if the item had a Random Suffix _(eq : Random Property)_. So this case shouldn't even exist

Reply:
wait you reviewed your own code?? :rofl:

Reply:
why not :)

Reply:
The deleted block was already there, this is not mine so hahaha
But only commented to say that even if my PR isn't merged, we need to remove this block because it's useless

Reply:
ok